uberon_edit.obo

```diff
@@ -91212,7 +91212,7 @@ xref: MA:0003122
 xref: MBA:10703
 intersection_of: UBERON:0002304 ! layer of dentate gyrus
 intersection_of: bounding_layer_of UBERON:0001885 ! dentate gyrus of hippocampal formation
-relationship: composed_primarily_of GO:0044464 ! cell part
+relationship: composed_primarily_of CL:0000000 ! cell
```

Comment:
this is no longer correct. The layer need not consist of whole cells.

While this could be represented by an owl class expression `composed_primarily_of some (part-of some cell)`...

the value of the axiom doesn't really warrant the complexity

remove it

for specific layers we can actually describe what the parts are - e..g synapses, somas, axons, ....